strawberryfields/backends/tfbackend/circuit.py

```diff
@@ -794,7 +794,7 @@ def measure_homodyne(self, phi, mode, select=None, **kwargs):
 
             self._update_state(new_state)
 
-        return meas_result
+        return tf.cast(meas_result, dtype=ops.def_type)
```

Comment:
@josh146 @ziofil Please make sure this suggestion makes it to the branch that is taking over

Reply:
Done!